acsoo/templates/project/+project.name+/.coveragerc.bob

```diff
@@ -0,0 +1,27 @@
+[paths]
+source =
+    odoo/addons
+    */{{{ project.name }}}/odoo/addons
+
+[run]
+dynamic_context = test_function
+
+[report]
+include =
+    */odoo/addons/*
+
+omit =
+    */scenario/*
+    */scenarios/*
+    */test/*
+    */tests/*
```

Comment:
why omit tests ?

Reply:
If we don't omit tests files, they will appear in coverage reports. But coverage of tests files is not relevant.

Reply:
I have seen cases where coverage of test files revealed issues where some tests weere not being executed do to bugs or configuration errors. What is the downside if we don't omit tests ?